cmd/start.go

```diff
@@ -110,6 +110,18 @@ type Start struct {
 	ZCash                string   `long:"zcash" description:"use a ZCash wallet in a dedicated data directory. To use this you must pass in the location of the zcashd binary."`
 }
 
+var (
+	mainConfig			[]byte
```

Comment:
`mainConfig` is actually data and unusable as a config in this state. Want to keep var names as accurate as possible with something like `configData` or similar.